python/pyspark/worker.py

```diff
@@ -2029,10 +2006,8 @@ def check_return_value(res):
                     if check_output_row_against_schema is not None:
```

Comment:
Let's also remove this block? It's not performant and not really needed to check against the row schema